configs/setr/setr_mla_vit-large_8x1_768x768_80k_cityscapes.py

```diff
@@ -0,0 +1,84 @@
+_base_ = [
```

Comment:
rename `configs/setr/setr_mla_vit-large_8x1_768x768_80k_cityscapes.py` to `configs/setr/setr_vit-large_mla_8x1_768x768_80k_cityscapes.py`